pyanaconda/dbus/__init__.py

```diff
@@ -0,0 +1,12 @@
+import os
+import pydbus
+
+def get_bus():
+    """Get a DBUS bus connection corresponding to the current environment.
+
+    During normal usage this function should return connection to the system bus,
+    but during testing/development a custom bus might be used.
+    So jut always connect to the bus specified by the DBUS_STARTER_ADDRESS
+    environmental variable.
+    """
+    return pydbus.connect(os.environ["DBUS_STARTER_ADDRESS"])
```

Comment:
AFAIK it should always be set for all DBUS started modules/services (so both for GTestDBus & normal system wide installed services/modules). I guess we can add a fallback to system bus just in case.